### ISSUE TITLE
Accuracy and Evaluation Scores not proportional

### ISSUE THREAD

**Issue by Verdia** [leondgarse/Keras_insightface#106]
Hello, thank you for your great works, i want to ask about the score of evaluation. In my training with 50 epoch, i got 100% accuracy and loss 0,0032%. But in evaluation, max of their scores very low. 

- max LFW i got 87,8%

- max CFP_FP i got 71,3%

- max AGEDB i got 60,7%

is there any problem with my training parameters? 

`from tensorflow import keras
import losses, train, models
import tensorflow_addons as tfa

basic_model = models.buildin_models("mobilenet", dropout=0, emb_shape=256, output_layer="GDC")

data_path = 'dataset/faces_emore_112x112_folders'
eval_paths = ['dataset/faces_emore/lfw.bin', 'dataset/faces_emore/cfp_fp.bin', 'dataset/faces_emore/agedb_30.bin']

tt = train.Train(data_path, save_path='keras_29122022_1445.h5', eval_paths=eval_paths,
                basic_model=basic_model, batch_size=512, random_status=0,
                lr_base=1e-4, lr_decay=0.001, lr_decay_steps=1e-5, lr_min=1e-3)

optimizer = tfa.optimizers.AdamW(learning_rate=0.001, weight_decay=5e-5)
sch = [
  {"loss": losses.ArcfaceLoss(scale=16), "epoch": 15, "optimizer": optimizer, "triplet": 64},
  {"loss": losses.ArcfaceLoss(scale=32), "epoch": 15},
  {"loss": losses.ArcfaceLoss(scale=64), "epoch": 20},
]
tt.train(sch, 0)`

i'm using datasets faces_emore (i only take 200 class from thousands class). I already use your training script in readme but the scores is lower than my parameter.

i also attach my json history in case you want to see it.
Thank you.
[keras_29122022_1445_hist.txt](https://github.com/leondgarse/Keras_insightface/files/10322755/keras_29122022_1445_hist.txt)

**Comment by leondgarse** [comment]
Sorry for the late reply. I've took some tests also using 200 classes, it's just too small a dataset that cannot reach a good result... I think there are some issues in your script:
- For learning_rate, setting `lr_min=1e-3` will force using `lr=1e-3` in the entire training process. Most time I'm using cosine decay, may try `lr_base=0.001, lr_decay=0.5, lr_decay_steps=49, lr_min=1e-4`.
- Using `"triplet"` is not recommended in early training steps, as the model still cannot select good positive and negative samples.

Anyway, I cannot train a better model using 200 classes either. You may try using the whole dataset, and should be better.

**Comment by Verdia** [comment]
Ahh.. i see.., thanks for the advice and your help, i will try with your parameters and whole dataset. For now, i'll close the issue. Thank you.. 🙏
